website/source/api/acl/identity-providers.html.md

```diff
@@ -0,0 +1,328 @@
+}
+```
+
+### Sample Request
+
+```text
+$ curl \
+    --request PUT \
+    --data @payload.json \
+    http://127.0.0.1:8500/v1/acl/idp/minikube
+```
+
+### Sample Response
+
+```json
+{
+    "Name": "minikube",
+    "Description": "updated name",
+    "Type": "kubernetes",
+    "KubernetesHost": "https://192.0.2.42:8443",
+    "KubernetesCACert": "-----BEGIN CERTIFICATE-----\nMIIC5zCCAc+gAwIBAgIBATANBgkqhkiG9w0BAQsFADAVMRMwEQYDVQQDEwptaW5p\na3ViZUNBMB4XDTE5MDEwODE4NTAwNFoXDTI5MDEwNjE4NTAwNFowFTETMBEGA1UE\nAxMKbWluaWt1YmVDQTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBALna\nHbH7BirRjzbExPgGdsikbPUtwmUqyGvJ5SGGho0RkkL/HIL5D154CD8CvAMEbSMt\n20FBqbI5h5JeJ9gYUdsPLwnMVI2r2yvJuzuPxtxiyf63DU7XxXi3v6QP6J0qRhDr\nlEhp5K2Xrd7qIxABZokNqPF8o5cfSjVgoREn6wKpmcCsf7rAqRpvMZWWYwsyAZJK\nEqWASwY6PmcwN3tboLoPBv7ZSJ8Q3y6RBypeNbngJgylNXP/OGwNO9Nm/HpmL1vc\nlR92K+tJE43XONkjkuXicC+ImhiX8cA6elmNIPog0UcpXal8CrDRug9MiGV3AJit\npxDjRFa163mjx536bE0CAwEAAaNCMEAwDgYDVR0PAQH/BAQDAgKkMB0GA1UdJQQW\nMBQGCCsGAQUFBwMCBggrBgEFBQcDATAPBgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3\nDQEBCwUAA4IBAQB+MRDedyRpsL08sDuU8EnJ/tEPZ6B7uTTBFrPyLoxbc7DfZEm0\nPlwyMk04TNlxoWLlcs4DBI/goVpER9kVoSd9CWmwG62Mqptk/AsyBlxqh0ZbHkAU\ndPHBXHFdoCkGnM//HDc8DW9zKGjv2wlKptPUxq1tkAX5aKUktsOVpmKH0YgKLuWA\n3Hvhgh2Gd/ssIwp1pHbWHKZ9+HS1etwHqgSneLpQ50K4iv0Rk7yxQmfCQC2wiz4q\n5T4dUXB28k0hl/Lx1QCfMvy20btyhnPleeW0xiQZ4Yq/XYv11Ez81duOKjZGYick\nZvw0ZvX68ssuWgbrWHkFSFJyBskBZOl1Rtln\n-----END CERTIFICATE-----\n",
+    "KubernetesServiceAccountJWT": "eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJkZWZhdWx0Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZWNyZXQubmFtZSI6ImNvbnN1bC1pZHAtdG9rZW4tcmV2aWV3LWFjY291bnQtdG9rZW4tbHBteHYiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC5uYW1lIjoiY29uc3VsLWlkcC10b2tlbi1yZXZpZXctYWNjb3VudCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6IjZiMGU4NzkxLTU3ZDItMTFlOS1iYzJhLTQ4ZTZjOGI4ZWNiNSIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDpkZWZhdWx0OmNvbnN1bC1pZHAtdG9rZW4tcmV2aWV3LWFjY291bnQifQ.bdhKU-r_5jJeg9kKIhqmZDxqSN4rD0wSvpDc_JBfP8WqLeaA7bZctt9oQW8FtNqMtlnM7eLsRkHspqNuudXADz5FWuWmU59jqccfrwazYm5qHpOWdYcVTZKeH1cCxheWW0GDuTPrXINcJoRFS6wxhytBLC-aBy3pObGjmEpApr3G9SAcY9cT5RgkglTx9yJhQHxIICHg9ktxPKy86WkaJq4k8L6lK-3oK0Ul6gy4Dgmx_DvgBrN_dAOmWWLX3nYUT62TpWLDto6wmXW9Z354ziv_XBlmq52K5lQw8pq1B7M8scUanSJ693CGippMbutjV3RJ08UkfgsN1DHWVUQ4pQ",
```

Comment:
I think we should truncate all these tokens and certificates. If necessary we can call that out at the beginning of the doc.

Reply:
I'll just do it how I noticed the vault docs do it: https://www.vaultproject.io/api/auth/kubernetes/index.html